ivy/functional/frontends/torch/tensor_functions.py

```diff
@@ -26,3 +26,17 @@ def is_nonzero(input):
 @to_ivy_arrays_and_back
 def is_complex(input):
     return ivy.is_complex_dtype(input)
+
+
+@to_ivy_arrays_and_back
+def scatter(input, dim, index, src, reduce=None):
+    return ivy.put_along_axis(input, index, src, dim, mode=reduce)
+
+
+@to_ivy_arrays_and_back
+def scatter_add(input, dim, index, src):
```

Comment:
We should probably include `scatter_add_` and `scatter_reduce_`. These are both placeholder functions but there may be cases where these are called by the GC.